### PR TITLE
Codechunks for returning outs, remove old Builder

### PR DESCRIPTION
**Pull request by EPashkin**
Part of #150.

### CODE REVIEW

**Comment by gkoz** [comment]
Good job!

File: src/chunk/chunk.rs [hunk 1]
```diff
@@ -11,8 +13,15 @@ pub enum Chunk {
     FfiCallTODO(String),
     FfiCall{name: String, params: Vec<Chunk>},
     FfiCallParameter{par: parameter_ffi_call_in::Parameter, upcast: bool},
+    FfiCallOutParameter{par: parameter_ffi_call_out::Parameter},
     //TODO: separate without return_value::Info
     FfiCallConversion{ret: return_value::Info, call: Box<Chunk>},
+    Let{name: String, is_mut: bool, value: Box<Chunk>},
+    Uninitialized,
+    VariableValue{name: String},
+    Tupple(Vec<Chunk>),
```
Comment (gkoz) [inline src/chunk/chunk.rs]: Tuple

Comment (EPashkin) [inline src/chunk/chunk.rs]: Oh, again :( I still misprint

**Comment by EPashkin** [comment]
Fixed

**Comment by gkoz** [comment]
Thanks.
@homu r+

**Pull request merged**